docs/user_guide/telemetry.md

```diff
@@ -0,0 +1,23 @@
+# Telemetry
+
+This library collects telemetry data by default. Telemetry events contain non-personally identifiable information that helps us understand how users interact with our software. We use this to know which features our customers use and/or what existing pain points are.
+
+You can opt out of telemetry on your machine by editing your Deadline Cloud configuration, or temporarily by setting an environment variable in your terminal.
+
+**To opt out of telemetry (config)**
+
+1. Run the `deadline config` command:
+```sh
+deadline config set telemetry.opt_out true
+```
```

Comment:
Telemetry opt out is also a checkbox in the submitter settings page I think. What about suggesting the GUI method instead here?